src/esp/gfx/PbrDrawable.cpp

```diff
@@ -25,48 +28,55 @@ PbrDrawable::PbrDrawable(scene::SceneNode& node,
       shaderManager_{shaderManager},
       lightSetup_{shaderManager.get<LightSetup>(lightSetupKey)},
       materialData_{
-          shaderManager.get<MaterialData, PbrMaterialData>(materialDataKey)},
+          shaderManager.get<Magnum::Trade::MaterialData>(materialDataKey)},
```

Comment:
```suggestion
          shaderManager.get<Mn::Trade::MaterialData>(materialDataKey)},
```

given you're doing that shortening cleanup anyway :D